libs/community/langchain_community/embeddings/fastembed.py

```diff
@@ -59,13 +58,13 @@ class Config:
     def validate_environment(cls, values: Dict) -> Dict:
         """Validate that FastEmbed has been installed."""
         try:
-            from fastembed.embedding import FlagEmbedding
```

Comment:
is it worth adding something to the ImportError below specifying the minimum expected version?

Reply:
Right. Will do.

Reply:
https://github.com/langchain-ai/langchain/commit/edf7b0aeae09adb16094bcf255543dc93ab0c3d9